package.json

```diff
@@ -12,6 +12,9 @@
     "prettier": "prettier '**/*.js' '**/*.md' '**/*.ts' --write",
     "release": "lerna publish"
   },
+  "dependencies": {
+    "extract-files": "^5.0.1"
```

Comment:
@bmullan91 I'm now wondering if I should have installed this in `graphql-hooks` specifically...

Reply:
Addressed in d96baff.